### PR TITLE
Added Github Page_build messege Handler

### PR DESCRIPTION
**Pull request by Ghost-script**
Added handler for github page build messages . Enhancement  #188

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/tests/github.py [hunk 1]
```diff
@@ -1834,6 +1834,112 @@ class TestGithubRelease(Base):
...
+class TestGithubPageBuild(Base):
+    """ There exists `a service
+    <https://apps.fedoraproject.org/github2fedmsg>`_ to link the select github
+    repos of fedora contributors with the fedmsg bus.
+
+    Messages of *this* type are published whenever someone **builds a Github Page**.
+    """
+    expected_title = "github.page_build"
+    expected_subti = None
+    expected_link = "https://github.com/fedora-infra/fas3-api-haskell"
```
Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/tests/github.py]: Ooo.. can you make the ``expected_link`` be ``"http://fedora-infra.github.io/fas3-api-haskell/"``?  That's the link to the page that actually got built.

File: fedmsg_meta_fedora_infrastructure/tests/github.py [hunk 2]
```diff
@@ -1834,6 +1834,112 @@ class TestGithubRelease(Base):
...
 
+class TestGithubPageBuild(Base):
+    """ There exists `a service
+    <https://apps.fedoraproject.org/github2fedmsg>`_ to link the select github
+    repos of fedora contributors with the fedmsg bus.
+
+    Messages of *this* type are published whenever someone **builds a Github Page**.
+    """
+    expected_title = "github.page_build"
+    expected_subti = None
```
Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/tests/github.py]: And for the expected subtitle here, can you make it something like ``"relrod rebuilt the github.io page for fedora-infra/fas3-api-haskell"``?

**Comment by ralphbean** [comment]
This is a really good start!  I left some inline feedback above.  Does it make sense?

**Comment by Ghost-script** [comment]
@ralphbean , i have implemented the above. Should it be **relrod** or **codeblock** in subtitle ( `relrod rebuilt the github.io page for fedora-infra/fas3-api-haskell` ) the _get_user() function returns **codeblock** . Any other modifications?  Any `expected_objects`?

**Comment by ralphbean** [comment]
> the _get_user() function returns codeblock

Ah, my mistake.  Yes.. it should be 'codeblock' here.

> Any expected_objects?

Oh, yes.  Add in ``'fedora-infra/fas3-api-haskell/page_build',`` there.

> should i update this pull request?

Yeah!  I'd love to see the changes.  :)

**Comment by ralphbean** [comment]
Looks good to me!  This fixes #188.

Thanks @Ghost-script!

**Pull request merged**